ui/app/templates/allocations/allocation/index.hbs

```diff
@@ -190,28 +213,39 @@
       <div class="boxed-section-body is-full-bleed">
         <ListTable @source={{this.ports}} as |t|>
           <t.head>
-            <th>Name</th>
-            <th>Host Address</th>
-            <th>Mapped Port</th>
+            <th>
+              Name
+            </th>
+            <th>
+              Host Address
+            </th>
+            <th>
+              Mapped Port
+            </th>
           </t.head>
           <t.body as |row|>
             <tr data-test-allocation-port>
-              <td data-test-allocation-port-name>{{row.model.label}}</td>
+              <td data-test-allocation-port-name>
+                {{row.model.label}}
+              </td>
               <td data-test-allocation-port-address>
                 <a
                   href="http://{{row.model.hostIp}}:{{row.model.value}}"
                   target="_blank"
                   rel="noopener noreferrer"
-                >{{row.model.hostIp}}:{{row.model.value}}</a>
+                >
+                  {{row.model.hostIp}}:{{row.model.value}}
```

Comment:
I reverted the change to use a helper here. It seemed like a good idea to add more good to workaround a tooling issue.

It also seems like Prettier is fine with the code like this? It didn't try to change it for me at least, so I think it may be another plugin in your machine that is causing this formatting.

Moving forwards though, I think we should reconsider using Prettier for `hbs` files. I detected some other minor visual regressions caused by all these formatting and I worry about having others we may have missed. I will leave this decision to you and @philrenaud.